kustomize/overlay/7.1.0/securebanking/devops/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: securebanking-platform-config
+data:
+  BASE_FQDN: devops.forgerock.financial
+  IG_FQDN: obdemo.devops.forgerock.financial
+  RS_FQDN: rs.devops.forgerock.financial
+  RCS_FQDN: rcs.devops.forgerock.financial
+  RCS_UI_FQDN: rcs-ui.devops.forgerock.financial
+  IDENTITY_PLATFORM_FQDN: iam.devops.forgerock.financial
```

Comment:
why is kustomize stuff being changed in this PR?

Reply:
Ah I must have not done a latest pull when creating the branch as this stuff is already in the master branch for the devops cluster

Reply:
That's now been sorted